roxie/ccd/ccdmain.cpp

```diff
@@ -1129,10 +1132,56 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                 if (port)
                 {
                     const char *protocol = roxieFarm.queryProp("@protocol");
+                    const char *passPhrase = nullptr;
```

Comment:
Sorry I meant the next two lines.. certFileBuf and keyFileBuf are what are used in the call to createListener.  I think the following to variables are only used if protocol == "SSL".